tldraw/packages/react/src/hooks/useCanvasEvents.ts

```diff
@@ -77,7 +77,10 @@ export function useCanvasEvents() {
       onDragOver,
       // fix touch callout in iOS
       onTouchEnd: (e: TouchEvent) => {
-        e.preventDefault()
+        let tool = app.selectedTool.id
+        if (tool === 'pencil' || tool === 'highlighter') {
```

Comment:
Previously, e.preventDefault will be triggered when you click any button on the whiteboard, so using the pencil to create logseq portals stopped working.

> I think we can also try to use pointerdown events to handling button clicks instead of real clicks for better pencil ux.

Will give this a try.

Reply:
I tried to move the handler to `onPointerDown`, but the callout magnifier problem shows again.